go.mod

```diff
@@ -33,7 +33,7 @@ require (
 	github.com/jarcoal/httpmock v1.3.1
 	github.com/joho/godotenv v1.5.1
 	github.com/jpillora/longestcommon v0.0.0-20161227235612-adb9d91ee629
-	github.com/keboola/go-client v1.22.0
+	github.com/keboola/go-client v1.22.1-0.20240316144146-9e0e53d37a51
 	github.com/keboola/go-utils v0.8.6
```

Comment:
Will be updated to `1.23.0`, after the release: https://github.com/keboola/go-client/pull/126